OI does not require sfw (and that is a 32-bit inclined path anyway)

--- nut-2.8.1-orig/m4/nut_check_libusb.m4	2023-09-19 23:43:43.000000000 +0200
+++ nut-2.8.1-fixed/m4/nut_check_libusb.m4	2023-11-04 21:06:37.924268597 +0100
@@ -309,7 +309,8 @@
 			[solaris2.1*], [
 				AC_MSG_CHECKING([for Solaris 10 / 11 specific configuration for usb drivers])
 				AC_SEARCH_LIBS(nanosleep, rt)
-				LIBS="-R/usr/sfw/lib ${LIBS}"
+				dnl #illumos is modern# LIBS="-R/usr/sfw/lib ${LIBS}"
+				LIBS="${LIBS}"
 				dnl FIXME: Sun's libusb doesn't support timeout (so blocks notification)
 				dnl and need to call libusb close upon reconnection
 				dnl TODO: Somehow test for susceptible versions?
